package.json

```diff
@@ -53,9 +53,11 @@
   },
   "browserslist": {
     "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
+      "chrome >= 67",
+      "edge >= 79",
+      "firefox >= 68",
+      "opera >= 54",
+      "safari >= 14"
     ],
     "development": [
       "last 1 chrome version",
```

Comment:
had a think about this more - i can't see a good reason why webpack/babel should target latest versions locally in development (other than less transpilation/higher perf) than what we run in prod

we want our dev environments to match prod as closely as possible. if this had always been the case, this would never have happened in the first place

any reason why we shouldn't have dev match production?

Reply:
also i'd like to

- remove opera and edge targeting, i haven't heard of a single crypto user using these?
- crank the chrome/firefox/safari versions higher, e.g. chrome is currently at 109 🤡